pkg/controller/backup_controller.go

```diff
@@ -250,6 +253,11 @@ func (c *backupController) prepareBackupRequest(backup *velerov1api.Backup) *pkg
 	// calculate expiration
 	if request.Spec.TTL.Duration > 0 {
 		request.Status.Expiration = metav1.NewTime(c.clock.Now().Add(request.Spec.TTL.Duration))
+	} else {
+		// set default backup TTL
```

Comment:
we should also set `request.Spec.TTL` here so that the API object gets patched with the default TTL. So, you might want to reorg L254-261 to first check if there's a TTL, and if not, assign it the default. Then, calculate the `Expiration`, whether there was a TTL specified or not.

Reply:
Sure, will do that.